Server/service/networkService.js

```
@@ -98,9 +99,19 @@ class NetworkService {
 				await monitor.save();
 
 				if (oldStatus !== undefined && oldStatus !== isAlive) {
-					this.handleNotification(monitor, isAlive);
+					const config = {
+						monitor: monitor,
+						template: isAlive === true ? "serverIsUpTemplate" : "serverIsDownTemplate",
+						context: { monitorName: monitor.name, monitorUrl: monitor.url },
+						subject:
+							(subject = `Monitor ${monitor.name} is ${isAlive === true ? "up" : "down"}`),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)